### ISSUE TITLE
Using default Fallout4.ini not MO2's

### ISSUE THREAD

**Issue by LumberingTroll** [TanninOne/modorganizer#689]
When launching through MO2.0.5 the game is using the default Fallout4.ini as a result not all of the mods are loading and causing issues, Tested and verified by adding access lines to the default Fallout4.ini located in /my game/fallout4 directory. The fallout4prefs.ini appears to be working correctly.

**Comment by KeltecRFB** [comment]
I am having this same issue hence the reason I consolidated from 688 into 689.

**Comment by bitdman** [comment]
_

> When launching through MO2.0.5 the game is using the default Fallout4.ini as a result not all of the mods are loading and causing issues, Tested and verified by adding access lines to the default Fallout4.ini located in /my game/fallout4 directory. The fallout4prefs.ini appears to be working correctly.__

_Was there a fallout4custom.ini in my doc.s FO4 folder with fallout4.ini and prefs.ini?

**Comment by LumberingTroll** [comment]
There was initially, however I deleted all ini files in the mydocks and tried again, it generated new ini files and did the same thing.

**Comment by YoungWaltDisney** [comment]
Mine has a fallout4custom.ini in fallout4 documents folder. Is that of any significance?

**Comment by bitdman** [comment]
It would have been if it was missing and the content from it is missing. When I install the game I no longer have the custom ini anymore but it appears everything is running fine.

**Comment by WhiteZeroX** [comment]
So this only happens when using the classic Portable install method? Doing the "Regular" install that uses appdata doesn't have this issue? Guess I might need to reinstall and try that.

**Comment by Chris-Barton** [comment]
I also have this issue however for me it loads all loose files as far as im aware but not any of the esp/esm's

**Comment by bitdman** [comment]
The only thing I see that is common for all three of these issues is that nobody has said anything about checking in the "PROFILE" folder where all the INI's, Loadorder.txt, mod list and plugin.txt for all the mods and saves that apply to that "PROFILE." Go to where you have MO installed, open the folder where your profiles are, open a profile folder and check on your folders and files I named above If they're not in there then something went wrong with the install.....I hope this helps,,,,I don't know of any other way to explain it

**Comment by WhiteZeroX** [comment]
@bitdman yep, I have all those files.

**Comment by bitdman** [comment]
can you explain again what is happening.....why do you think it's not loading textures and meshes, or esp's? there is a mod list ,and a plugin .txt in the profile folder? correct?

**Comment by WhiteZeroX** [comment]
@bitdman no, in my Issue #697 I described that loose files such as textures and whatnot DO load, but no ESPs are loading for me. If I load a Save game that has ESPs, I get the warning that they are no longer present.

**Comment by bitdman** [comment]
But that message indicates that you removed mods from that save
are you adding mods to a save game or removing?
If you try to load a profile that has mods that have not yet been installed
even if you have the textures and meshes installed it can't activate the
mod withou the esp

**Comment by WhiteZeroX** [comment]
@bitdman Exactly, it thinks I removed mods because MO is not loading them. I'm not adding or removing anything, just using my mods/profiles and saves that worked in previous versions. And yes, I've tried brand new profiles and reloading all my mods.

**Comment by WhiteZeroX** [comment]
I'd just like to add that when I launch LOOT through MO, all of my ESP do show up there, even though they aren't loading inside of FO4.

**Comment by Willlux** [comment]
Still experiencing this issue with 2.0.6b
All of the relevant .ini files are in the profile folder, but the game uses the default .ini's regardless.

**Comment by KeltecRFB** [comment]
This still seems to be a problem in 2.0.6 (Beta 2).  It is reading some other INI (Default perhaps) and not reading the profiles INIs.

**Comment by TanninOne** [comment]
Could someone affected please attach the usvfs_<date>.log from a game-run?

**Comment by WhiteZeroX** [comment]
@TanninOne Here ya go. Logs don't show any .ini's being touched. 
[usvfs-2016-06-05_13-25-06.zip](https://github.com/TanninOne/modorganizer/files/299347/usvfs-2016-06-05_13-25-06.zip)

**Comment by KeltecRFB** [comment]
Sorry, I will reinstall 2.0.6, run the game, and post the usvfs logs.

**Comment by KeltecRFB** [comment]
With 2.0.6 running, old saves pre-2.0.6 disappear and all profile INI adjustments I have made for mods no longer work.  

[usvfs-2016-06-05_16-12-50.zip](https://github.com/TanninOne/modorganizer/files/299400/usvfs-2016-06-05_16-12-50.zip)

**Comment by KeltecRFB** [comment]
With 2.0.4 running, old saves restored and INIs are working.  Of course, I have to use a mod that adds the asterisks to the plugins in order for the plugins to load.

[usvfs-2016-06-05_16-26-18.zip](https://github.com/TanninOne/modorganizer/files/299408/usvfs-2016-06-05_16-26-18.zip)

**Comment by LoCInuyasha** [comment]
Hopefully someone will appreciate my signing up for this site solely to share the answer..   But after searching for a while online, this was all I could find, with no answers...

The problem's in the .ini files.  fallout4.ini sets:
SLocalSavePath=__MO_Saves\

But then, fallout4custom.ini resets that same path back to Saves\

Update fallout4custom.ini and you'll be golden.
